src/sysusers/sysusers.c

```diff
@@ -801,7 +803,7 @@ static int write_files(void) {
         return 0;
 }
 
-static int uid_is_ok(uid_t uid, const char *name) {
+static int uid_is_ok(uid_t uid, const char *name, bool skip_gid_reuse_check) {
```

Comment:
I prefer here to invert the value and change the name e.g. `check_with_gid` or something. We usually do not name boolean value with negative words. `skip` is almost equivalent to `no` here.